dcpy/models/file.py

```diff
@@ -56,12 +55,19 @@ class Geodatabase(BaseModel, extra="forbid"):
     crs: str
 
 
-# TODO: implement JSON and GEOJSON
 class Json(BaseModel):
```

Comment:
Also, we should make sure these all have `extra="forbid"` (my bad - I added this one)